src/actions/sources/select.js

```diff
@@ -79,7 +84,7 @@ export function selectSourceURL(
  * @memberof actions/sources
  * @static
  */
-export function selectSource(sourceId: string, tabIndex: string = "") {
+export function selectSource(sourceId: string, tabIndex: number = undefined) {
```

Comment:
No real point in an `undefined` default, that would be the case if you just left it off.